runtime/expr/agg/ztests/collect-untag-unions.yaml

```
@@ -0,0 +1,9 @@
+zed: collect(this)
+
+input: |
+  1((int64,string))
+  "foo"((time,string))
+  .5
```

Nit: Make this canonical.
```suggestion
  0.5
```